CHANGES.rst

```diff
@@ -3,9 +3,12 @@ Changes
 
 .. currentmodule:: objgraph
 
-1.8.2 (unreleased)
+1.9 (unreleased)
 ------------------
```

Comment:
1.9.0 would match the style of previous releases -- I see 1.8.0, 1.7.0 etc. in this file.